src/lib/client.js

```diff
@@ -25,7 +25,10 @@ class Client extends EventEmitter {
       throw new TypeError('Client pluginOpts must be an object')
     }
 
-    if (typeof pluginOpts._plugin !== 'function') {
+    // if pluginOpts is an eventEmitter, then it is instantiated
+    const instantiated = (typeof pluginOpts.on === 'function')
+
+    if (!instantiated && typeof pluginOpts._plugin !== 'function') {
       throw new TypeError('"pluginOpts._plugin" must be a function')
```

Comment:
Update error message (and jsdoc) to reflect that it must be a function _or_ a plugin instance.